cmd/operator/main.go

```diff
@@ -117,6 +117,7 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.Var(&devices, "devices", "Device(s) to set up.")
+	flag.BoolVar(&qat.Unconfined, "qat-unconfined", true, "Run qat plugin in unconfined mode.")
```

Comment:
the operator flags should not include any device specific parameters. If we must have it, `QatDevicePlugin` API would have to be changed but we can probably keep the configurability using the existing CRD annotations flow.